coralproject/talk
# [CORL-1131] Add feature flag to disable live updates at tenant level

## What does this PR do?

Adds `DISABLE_LIVE_UPDATES` feature flag.

## What changes to the GraphQL/Database Schema does this PR introduce?

Adds `DISABLE_LIVE_UPDATES` feature flag.

## How do I test this PR?

- Enable `DISABLE_LIVE_UPDATES`
- Check admin that the option is no longer available under `Config > Advanced`
- See live updates aren't showing on comment stream
